store/rootmulti/store.go

```diff
@@ -763,8 +767,14 @@ func (rs *Store) Snapshot(height uint64, protoWriter protoio.Writer) error {
 	// and the following messages contain a SnapshotNode (i.e. an ExportNode). Store changes
 	// are demarcated by new SnapshotStore items.
 	for _, store := range stores {
+		rs.logger.Debug("starting snapshot", "store", store.name, "height", height)
 		exporter, err := store.Export(int64(height))
+		if exporter == nil {
```

Comment:
Same comment that here: https://github.com/cosmos/cosmos-sdk/pull/13935/files#r1126755652